[Alerting V2][Serverless & 9.5][M2] Add experimental alerting features alerts pages (#6527)
elastic/docs-content: explore-analyze/alerting/kibana-alerting-experimental/alerts/triage-alert-episodes.md

---
navigation_title: Triage alert episodes
applies_to:
  stack: experimental 9.5+
  serverless: experimental
products:
  - id: kibana
description: "Take triage actions on alert episodes in Kibana's experimental alerting system. Acknowledge, snooze, resolve, activate, tag, and assign episodes individually or in bulk."
---

# Triage alert episodes [triage-alert-episodes]

In the {{alerting-v2-system}}, you can take the following triage actions on alert episodes individually or in bulk. For deeper investigation of a specific episode, refer to [Investigate alert episodes](investigate-alert-episodes.md).

## Available actions [row-level-actions]

- **Acknowledge / Unacknowledge:** Marks the episode as seen. Applies to the individual episode.
- **Snooze / Unsnooze:** Suppresses notifications for the episode's series for a set duration. Applies to all episodes sharing the same `group_hash`.
- **Resolve / Unresolve:** Closes the episode. Applies to the episode's series.
- **Activate:** Manually moves the episode to `active` state without waiting to meet the activation threshold.

> - **Edit tags:** Adds or removes tags on the episode.

Author: @adcoelho @umbopepato [there's an open question](https://github.com/elastic/docs-content/pull/5528/changes#r3229276705) about whether the edit tag action only applies to individual episodes or if it applies to every episode in the same series (a group).

Reviewer: This applies to all episodes sharing the same `group_hash`.

Author: Fixed in [`3f80ed4` (this PR)](https://github.com/elastic/docs-content/pull/6527/commits/3f80ed45d8929e2b3d73fd2c89f60f0ca822d7eb).

- **Assign:** Assigns the episode to a specific user. Available from the episode detail page.

## Bulk actions [bulk-actions]

All available actions apply to multiple episodes at once. Scope rules still apply: acknowledge and activate apply per episode, while snooze and resolve apply per series. When snoozing episodes across different series, the system snoozes each series independently.

## Snooze [snooze-episode]

> Snoozing suppresses notifications for an alert series for a defined period. The rule continues to evaluate and the episode remains visible, but {{kib}} sends no notifications until the snooze expires.

Author: Addressing Tia's feedback [here](https://github.com/elastic/docs-content/pull/5528/changes#r3229253632).

```suggestion
Snoozing suppresses actions from being triggered for an alert series for a defined period. The rule continues to evaluate and the episode remains visible, but actions aren't executed until the snooze expires.
```
